setup.py

```diff
@@ -1,16 +1,24 @@
-from __future__ import absolute_import
+import os
+import re
 
+from setuptools import find_packages, setup
 
-from setuptools import setup, find_packages
+
+def get_version():
```

Comment:
this seems way more complicated than the old version number scheme. what's the benefit?

Reply:
being able to get fastestimator version by fe.__version__